components/scream/src/physics/p3/tests/p3_unit_tests.cpp

```diff
@@ -80,6 +80,127 @@ struct UnitWrap::UnitTest<D>::TestP3Func
   }
 };
 
+
+template <typename D>
+struct UnitWrap::UnitTest<D>::TestP3CloudWaterAutoconversion
+{
+
+static void  cloud_water_autoconversion_unit_bfb_tests(){
+  using KTH = KokkosTypes<HostDevice>;
+
+  static constexpr Int max_pack_size = 16;
+  REQUIRE(Spack::n <= max_pack_size);
+
+  CloudWaterAutoconversionData cwadc[max_pack_size] = {
+    { 0.97026902585098274, 5.1000000000000004e-3, 206128398.07453227,  0.0, 0.0, 0.0},
```

Comment:
You don't need to explicitly initialize the outputs to 0.0, that's the default behavior.

Reply:
I removed the explicit initialization of 0.0. Thanks for catching this!